eng/common/scripts/New-ReleaseAsset.ps1

```diff
@@ -1,62 +0,0 @@
-<#
```

Comment:
Lets not remove this as part of this PR. If you want to remove it lets do it from the tools repo.

Reply:
On further discussion we don't need to remove this from eng/common. So exclude this file and keep it in eng/common because we may need it for something else.